### PR TITLE
Camera permissions

### LINKED ISSUE #285: Improve Camera Permission handling
CameraButton needs some improvement. It should store whether the user has approved CameraPermission before. Android probably doesn't need the Alert, as they can request permissions an infinite amount of times. iOS supposedly crash if you request after having been denied, so the handling of that should reflect the code.

There's some old code that can be removed.

### PR DESCRIPTION
**Pull request by Alexander-Mjoberg**
## Change description

Adds a redux state to keep track of if the user has accepted giving camera permission. If they click yes on the alert, they won't be prompted again about permission. 

## How to verify

1. Log on as company
2. Click + for Camera and accept both dialogues. 
3. Go back and check that you don't get asked about permissions but go straight to camera when clicking +

## Issues fixed

#285

### CODE REVIEW

File: src/components/CameraButton.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import {View, TouchableOpacity, Text, StyleSheet, PermissionsAndroid, Alert,Platform } from 'react-native'
```
Comment (arpi09) [inline src/components/CameraButton.js]: space after ","

File: src/screens/Profile/CameraScreen.js [hunk 2]
```diff
@@ -1,8 +1,7 @@
 import React, { Component } from 'react';
-//import { Text, View } from 'react-native';
 import PropTypes from 'prop-types'
 import { CameraKitCameraScreen } from 'react-native-camera-kit';
-import {View, Text,  StyleSheet, PermissionsAndroid, Alert,Platform } from 'react-native';
```
Comment (arpi09) [inline src/screens/Profile/CameraScreen.js]: Space after "," (I am a mean man).

File: src/screens/Profile/CameraScreen.js [hunk 3]
```diff
@@ -1,8 +1,7 @@
...
+import {View, Text,  StyleSheet } from 'react-native';
```
Comment (arpi09) [inline src/screens/Profile/CameraScreen.js]: Remove one space after "Text,"

File: src/screens/Profile/CameraScreen.js [hunk 4]
```diff
@@ -13,8 +12,8 @@ class CameraScreen extends Component {
 
 
 render() {
-  
-  return( 
+
```
Comment (arpi09) [inline src/screens/Profile/CameraScreen.js]: Remove this line

File: src/screens/Profile/CameraScreen.js (hunk 2 above)
Comment (Alexander-Mjoberg) [reply src/screens/Profile/CameraScreen.js]: Tror den här ska vara fixad om jag fattat det rätt 💃

**Pull request merged**